client/blocks/product-purchase-features/product-purchase-features-list/live-courses.jsx

```diff
@@ -7,9 +7,14 @@ import { localize } from 'i18n-calypso';
 /**
  * Internal dependencies
  */
+import analytics from 'lib/analytics';
 import PurchaseDetail from 'components/purchase-detail';
 import support from 'lib/url/support';
 
+function trackCoursesButtonClick() {
+	analytics.tracks.recordEvent( 'calypso_my_plan_courses_click' );
```

Comment:
I'd expect something like `calypso_purchase_features_courses_button_click`. Sounds kind of long but I only bring it up because I'm not sure where the `my_plan` part comes from. I'm sure there's something better than what I'm suggesting if what you have isn't already. The name here could get tricky because I'm sure at some point if we start tracking clicks for other purchase feature list items we'll want to group them together in a nice way.